src/bindings/python/src/pyopenvino/core/tensor.cpp

```diff
@@ -247,6 +255,25 @@ void regclass_Tensor(py::module m) {
             Sets Tensor's shape.
         )");
 
+    cls.def("is_continuous",
+            &ov::Tensor::is_continuous,
```

Comment:
Are there any tests for it?

Reply:
Let's remove this function and add it if necessary (request is coming).

Reply:
removed